src/components/video-texture-target.js

```diff
@@ -153,9 +153,8 @@ AFRAME.registerComponent("video-texture-target", {
         const texture = videoTextureSource.renderTarget.texture;
         this.applyTexture(texture);
 
-        // Bit of a hack here to only update the renderTarget when the screens are in view
-        material.map.isVideoTexture = true;
-        material.map.update = () => {
+        // Only update the renderTarget when the screens are in view
+        material.onBeforeRender = () => {
```

Comment:
Note to reviews: I assume `material.onBeforeRender` isn't set anywhere else.

Reply:
Oh wow I didn't realize this existed for materials. Well that makes things simple. We should probably use that for camera tool too then.

Reply:
Sounds good to me. Opened #5607